PeerioTechnologies/peerio-desktop
# fix: menus on macOS

"View" item was added in the dev build, but not on release, but macOS code relied on the order of menus to add items. Refactored to use variables.

#### Relevant info and issue/PR links 
 
https://app.clubhouse.io/peerio/story/9090/macos-help-menu-contains-items-from-window-menu

#### Testing instructions  

macOS only. In non-dev build, check that the main menu contain all the appropriate items and subitems.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
